gradle/dependencies.gradle

```diff
@@ -103,7 +103,7 @@ versions += [
   mavenArtifact: "3.6.3",
   metrics: "2.2.0",
   mockito: "3.2.4",
-  netty: "4.1.50.Final",
+  netty: "4.1.61.Final",
```

Comment:
Can we use latest `4.1.62.Final` release?

Reply:
Make sense.